_posts/2020-09-20-Linux-Install-Script.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "Linux Bash Installation Script"
+heading: "Bash Installation Script"
+author: "niebert"
+lang: "en"
+---
+
+Thanks to niebert, if you plan to install Jamulus on many Linux machines, you can try this script.
```

Comment:
What about a link for niebert here?
`[niebert](https://github.com/niebert)`

Reply:
added